src/main/java/com/exasol/repository/GitRepository.java

```diff
@@ -25,7 +25,8 @@
      * Get a new instance of {@link GitBranchContent} based on a user-specified branch.
      *
      * @param branchName name of a branch to get content from
-     * @return new instance of {@link GitBranchContent}
+     * @return new instance of {@link GitBranchContent}d
```

Comment:
```suggestion
     * @return new instance of {@link GitBranchContent}
```
Accidental key stroke?

Reply:
Probably :)